### ISSUE TITLE
marker:red when opening_hours not now or access no

### ISSUE THREAD

**Issue by chnuessli** [chnuessli/defikarte.ch-app#29]
can we do that?
if yes, then we should keep the marker dynamic and delete #23

**Comment by elektrolytmangel** [comment]
Yes we can try to do that. With the other JS-library there is a function to determine if it is open at the current time. We can make the marker dynamic eighter with a rendered view or with an image, this is not directly related. But close the #23, this is fine for me.

**Comment by elektrolytmangel** [comment]
We can add this to the first realease. @chnuessli you have to define how it should look like. Do we then still need that we can see if it is 24/7 or not? Probably we can show this on the pop-up / detail-map-overlay?

**Comment by chnuessli** [comment]
The marker should be as this:

- green: 24/7h or opening_hours right now opened
- yellow: access=private or opening_hours not known
- red: access=no or opening_hours right now closed

the same thing i want to add in the website as well, maybe i can take a look at your code afterwards :)
